src/containers/shared/utils.js

```diff
@@ -21,6 +21,29 @@ export const NOT_FOUND = 404
 export const SERVER_ERROR = 500
 export const BAD_REQUEST = 400
 
+export const FETCH_INTERVAL = 5000
+
+export const CHART_HEIGHT = 532
+export const CHART_MARGIN = { top: 5, right: 20, bottom: 5, left: 0 }
+export const CHART_YAXIS_LABEL = {
+  angle: -90,
+  position: 'insideTop',
+  dx: 45,
+  dy: 65,
+}
+export const CHART_XAXIS = {
+  angle: -65,
+  dy: 30,
+  dx: -7,
+  height: 90,
+  tickLine: false,
+  minTickGap: -1,
+  interval: 0,
+}
+export const BAR_SIZE = 30
+export const BAR_RADIUS = [4, 4, 0, 0]
+export const BAR_TOOLTIPS = { cursor: false, posX: 45, posY: 70, offset: -10 }
+
```

Comment:
Since these constants aren't used in any other files, I wouldn't store them in the `shared/utils.js` - It's cleaner in my opinion to keep them in the same file they're used, and if they are eventually needed elsewhere we can move them up then :)

Reply:
(With my previous comment about raising things up - I was mostly talking about if there were specific numbers which had to stay in sync within these - such as dx/posX since they seem to have the same number it'd be worth having a variable to guarantee that. If they just happen to be the same though it is not required)

Reply:
Oh I thought you meant those can be recycled for future charts. Yes it is just for styling and positioning this specific chart. I will just keep those numbers there like it was before and we can move some up in the future when new charts are in (Caleb have a stacked bar chart in mind now).